composer.json

```diff
@@ -22,8 +22,9 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
-        "charcoal/charcoal": "^2.0",
-        "mustache/mustache": "^2.14"
+        "charcoal/charcoal": "dev-feature/twig-implementation",
```

Comment:
Now that charcoalphp/charcoal#44 is merged, the Composer manifest and lock files can be updated.